pygmt/src/wiggle.py

```diff
@@ -23,10 +23,13 @@
     c="panel",
     d="nodata",
     e="find",
+    f="coltypes",
     g="gap",
     h="header",
     i="columns",
```

Comment:
`i` should be `incols` rather than `columns`. Would you be willing to open a PR similar to https://github.com/GenericMappingTools/pygmt/pull/1306 for this?

Reply:
Opened a corresponding PR in #1504.

Reply:
#1504 should be merged before this PR.